Xamarin.Forms.Platform.UAP/ImageButtonRenderer.cs

```diff
@@ -112,7 +112,9 @@ protected async override void OnElementChanged(ElementChangedEventArgs<ImageButt
 
 				if (Element.IsSet(ImageButton.CornerRadiusProperty) && Element.CornerRadius != (int)ImageButton.CornerRadiusProperty.DefaultValue)
 					UpdateBorderRadius();
-				if (Element.IsSet(Button.PaddingProperty) && Element.Padding != (Thickness)Button.PaddingProperty.DefaultValue)
+
+				// By default Button loads width padding 8, 4, 8 ,4
+				if (Element.IsSet(Button.PaddingProperty))
```

Comment:
If you try to clear the Padding property later, it will not clear the padding because of this check.

Reply:
nm